data/elevation-and-terrain/2020-lidar-northern-central-utah/index.md

```diff
@@ -0,0 +1,44 @@
+---
+status: publish
+layout: page
+tags:
+  - dem
+  - lidar
+  - elevation
+author:
+  display_name: Rick Kelson
+  email: rkelson@utah.gov
+published: true
+date: 2020-07-05 12:30:30
+title: 2020 Northern & Central Utah Lidar Elevation Data
+categories: []
+---
+
+![Sample]({% link images/lidar_northern_central_utah_2020.png %}){: .flex .flex--center .outline }
+
+[![Project area map]({% link images/lidar_northern_central_utah_2020_coverage_sm.png %} "click for map")]({% link images/lidar_northern_central_utah_2020_coverage.png %}){:.inline-text-right}
+
+{%- capture contact -%}
+{%- include contact.html subject=page.title contact=site.data.contacts.elevation text='or by request from' -%}
+{%- endcapture -%}
+During 2020 UGRC, the Natural Resource Conservation Service (NRCS), and the Utah Division of Emergency Management acquired [~5,262 square miles]({% link images/lidar_northern_central_utah_2020_coverage.png %}) of 2 and 8 points per meter Quality Level 1 & 2 LiDAR of portions of Northern and Central Utah. The .5 and 1 meter resolution bare earth DEMs and first-return/highest-hit DSMs in .tif format have a 10.0cm vertical RMSE accuracy and are available for download. The LAS point clouds are available from [The National Map](https://apps.nationalmap.gov/downloader) {{ contact }} This elevation data was collected between April and July, 2020.
```

Comment:
**suggestion**: put file format in code block
**suggestion**: say where to download the tifs from

```suggestion
During 2020 UGRC, the Natural Resource Conservation Service (NRCS), and the Utah Division of Emergency Management acquired [~5,262 square miles]({% link images/lidar_northern_central_utah_2020_coverage.png %}) of 2 and 8 points per meter Quality Level 1 & 2 LiDAR of portions of Northern and Central Utah. The .5 and 1 meter resolution bare earth DEMs and first-return/highest-hit DSMs in `.tif` format have a 10.0cm vertical RMSE accuracy and are available for download from the links below. The LAS point clouds are available from [The National Map](https://apps.nationalmap.gov/downloader) {{ contact }} This elevation data was collected between April and July, 2020.
```

Reply:
lets pass on this for now to keep the pages the same